pkg/vulnsrc/openeuler/openeuler.go

```diff
@@ -0,0 +1,246 @@
+package openeuler
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"path/filepath"
+	"strings"
+
+	bolt "go.etcd.io/bbolt"
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/trivy-db/pkg/db"
+	"github.com/aquasecurity/trivy-db/pkg/types"
+	"github.com/aquasecurity/trivy-db/pkg/utils"
+	"github.com/aquasecurity/trivy-db/pkg/vulnsrc/vulnerability"
+)
+
+type Distribution int
+
+const OpenEulerFormat = "openEuler-%s"
```

Comment:
We do not use this constant outside of the package.
```suggestion
const openEulerFormat = "openEuler-%s"
```